src/index.ts

```diff
@@ -196,6 +196,20 @@ function barfTurn() {
       if (!have($effect`Fishy`)) use($item`fishy pipe`);
     } else if (!embezzlerUp && timeToMeatify()) {
       useFamiliar($familiar`Grey Goose`);
+    } else if (
+      !embezzlerUp &&
+      have($familiar`Space Jellyfish`) &&
+      get(`_spaceJellyfishDrops`) < 5 &&
+      clamp(
+        myAdventures() - digitizedMonstersRemaining(),
+        globalOptions.saveTurns,
+        myAdventures()
+      ) -
+        globalOptions.saveTurns <
+        20 &&
```

Comment:
I might be about to call the kettle black here, but this math feels really hard to follow--could we get some comments and/or slicing it out into a function?

Reply:
Upon second thoughts the clamp() is not really needed, since I can accept negative values on the LHS just fine. That should simplify things greatly.